workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+| __Time Utilization__      | Displays total amount of time metric is in use |
+| __Count__                 | Displays number of times a metric has been queried over time |
+| __Age__                   | Displays the age of the `timestamp` value |
+
+#### Alias
+
+The Alias feature allows you to overwrite the metric name displayed in your widget for increased readability:
+
+![](https://lh7-us.googleusercontent.com/8gAYYTNk5B9dH_6a6S6DLq-zItBxQU_pv0GBuM0PnOonqY9kbxMiESmKRNa_vG6R7vDrhl5ELX_nRGloIlsHm5VDBM1xRuUuO7Az8wZk3orJnG6yrvTGeA8Nul_XDGugYy4_NnEavJISDxFtMdcMVyw)
+
+### Display Preferences
+
+The Display Preferences section is where you can specify the default time range that is displayed when you open the dashboard.
+
+<Note> If you specify a default time range, you are still able to drill into other time ranges with the Date Range filter. </Note>
+
+#### Name Your Widget
+
+The Widget Title is the title that will be displayed at the top of your widget on the dashboard.
+
+# Clone Dashboards & Widgets
+
+The clone feature allows you to experiment with new or slightly modified dashboard designs without disrupting your existing setup. Save time by cloning widgets instead of recreating them from scratch. 
+
+To clone a widget, click on the ellipses in the top right corner of the widget and select `clone`:
+
+![](https://lh7-us.googleusercontent.com/Y5euVq5Qju0bzuDRpwUpM1_8TFsWda4wL4WlRvnyYYYPP0DoLbrIfIE7_uNeRiqz6le4qI1EjkO000QEeBLNduUNDHrZY0oIs2C1Oi4bSYzhG2SvEvJDSobjr6U8IC3-kzwzbRj4Yfm7xsxJ0Q4wQjo)
+
+To clone a dashboard, click on the ellipses located in the dashboards table under __Actions__ and select __Clone__:
+
+![](https://lh7-us.googleusercontent.com/fMghA-ye5DNOrPlybyWCJhWBOGlPHJInIhX9uwiUnHJR42_ZMSeOeoG6w75rh6AQCW3rbG-sIKStnHSoW4zjISQwwy6GqzPxAd5z6Dv-8Fa86iUpsOvBrjCM3TU-Xuh_Bs1InLwtQqvcL_j06m_u80s)
+
+# Filter Dashboard Data
+
+### Live & History View
+
+View your static dashboard data in __History__ mode and view your real-time data in __Live__ mode. Toggle __History__ and __Live__ mode on and off by clicking the button in the top right corner of your dashboard:
+
+![](https://lh7-us.googleusercontent.com/hKkWW0kLLmlylBPMKA41MapsPsBS7qreD1td8lp9bUqHbyJM9GDk4azYn2eAM450PhFjET7UiCnu5Oq3dw3VBC-8ngkj_PeiRuf-lbFNp-YH1c2Qu-jVVRuOaytSOKDUasp6YNey7fYHG7N3rQMp0ZA)
+
+### Date Range
+
+The Date Range picker allows you to select the exact period of time to view your dashboard data. Choose predefined date ranges or select a custom range of days:
+
+![](https://lh7-us.googleusercontent.com/254d3LE1oktWBopnd4KekdbTlukBMOJAVLnZiErOQRh0E7cK33upT8JbR0iGaBvFTNozYXu-OEzEyViCmSKl6A-Y8FOutE5Okom7mvDXDPmEQOorSSR3sJJNS0uyXTjOMmDoKQ0Cn6ijngCFAeZVKN4)
+
+### Filters
+
+Filter metrics across your entire dashboard. The list of available filters is automatically built by the metadata pulled from all metrics being queried on the dashboard. Visualize the performance of one or multiple metrics within your broader dashboard by toggling on and off individual metrics:
```

Comment:
I would sub the last sentence as "Simply toggle on and off filters whenever you want to use them."